idaes/tests/test_config.py

```diff
@@ -71,3 +72,23 @@ def test_canonical_distro():
     assert idaes.config.canonical_distro("Ubuntu1804")  == "ubuntu1804"
     assert idaes.config.canonical_distro("Windows")  == "windows"
     assert idaes.config.canonical_distro("daRwin")  == "darwin"
+
+@pytest.mark.unit
+def test_warning_to_except():
+    with idaes.temporary_config_ctx():
+        with pytest.raises(RuntimeError):
+            _log = logging.getLogger("idaes")
+            idaes.cfg.warning_to_exception = True
+            idaes.reconfig()
+            _log.warning("Hey! Don't do that.")
+    _log.warning("Hey! Don't do that.")
```

Comment:
Yeah.  I did intend to put it there to make sure the config changes revert after leaving the context manager block.  Now that you mention it though, if we do use these to raise an error for warnings or deprecation warnings, the tests would fail.  I test the context manager elsewhere, so I guess I may as well take them out and assume it works to prevent potential future issues.